modules/system/console/MixCompile.php

```diff
@@ -197,7 +199,7 @@ protected function createWebpackConfig(string $mixJsPath): void
 
         $config = str_replace(
             ['%base%', '%notificationInject%', '%mixConfigPath%', '%pluginsPath%', '%appPath%'],
-            [$basePath, '', $mixJsPath, plugins_path(), base_path()],
+            [addslashes($basePath), '', addslashes($mixJsPath), addslashes(plugins_path()), addslashes(base_path())],
```

Comment:
Why is this required?

Reply:
While resolving the path, somehow php removes the slashes and it causes the following error in mix:

Error: Cannot find module 'D:DevWintercmsfixed-test/node_modules/laravel-mix/src/Engine'

Addslashes fixes the previously removed slashes.